third_party/3/orjson.pyi

```diff
@@ -2,16 +2,23 @@ from typing import Any, Callable, Optional, Union
 
 __version__ = str
```

Comment:
Surely this should be `__version__: str`.

Reply:
Not a new bug, but I sent a PR to ijl/orjson#119.